test/chpldoc/multifile.doc.compopts

```diff
@@ -0,0 +1 @@
+--process-used-modules
```

Comment:
Since the functionality of this test depends on a flag now, I'd be tempted to move it and its supporting .chpl file into the test/chpldoc/compflags directory for clarity.  Completely up to you, though.